src/components/AvatarImage/AvatarImage.module.css

```
@@ -17,17 +21,17 @@
   border-radius: var(--eds-border-radius-round);
 }
 
-/* TODO */
+/* 2 */
```

Here you can just say what this element is
```suggestion
/* Head of the avatar image. */
```
though I dunno if it's really necessary since the class names are pretty self-explanatory (we're just translating the class name to standard English).

---

That makes sense, will update the comments.